azure-pipelines-20230430.yml

```diff
@@ -292,9 +292,6 @@ stages:
                 && mvn clean install $(MVN_OPTS_INSTALL) -Phudi-platform-service -Pthrift-gen-source -pl hudi-utilities -am
                 && mvn test  $(MVN_OPTS_TEST) -Punit-tests -Dtest="Test*DeltaStreamer*" -DfailIfNoTests=false -pl hudi-utilities
                 && mvn test  $(MVN_OPTS_TEST) -Pfunctional-tests -Dtest="Test*DeltaStreamer*" -DfailIfNoTests=false -pl hudi-utilities
-                && ./scripts/ci/move_surefire_reports.sh /hudi /hudi/scripts/ci/results
-                && echo 'All surefire report files:'
-                && find . -type f -name \"TEST-*.xml\""
```

Comment:
The search folder `searchFolder` of the task `PublishTestResults@2` should change from `'$(Build.SourcesDirectory)/scripts/ci/results'` to `'$(Build.SourcesDirectory)'`

Reply:
Similar for `Top 100 long-running testcases`

Reply:
yeah, that's true